spec/requests/notifications_controller_spec.rb

```diff
@@ -288,6 +288,80 @@ def delete_notification(resp_code, matcher)
           delete_notification(403, :to)
         end
       end
+
+      describe '#mark_read' do
+        context "when targeting a notification by id" do
+          it 'can mark a notification as read' do
+            expect {
+              put "/notifications/mark-read.json", params: { id: notification.id }
+              expect(response.status).to eq(200)
+              notification.reload
+            }.to change { notification.read }.from(false).to(true)
+          end
+
+          it "doesn't mark a notification of another user as read" do
+            notification.update!(user_id: Fabricate(:user).id, read: false)
+            expect {
+              put "/notifications/mark-read.json", params: { id: notification.id }
```

Comment:
```suggestion
              put "/notifications/mark-read.json", params: { id: notification.id }
              expect(response.status).to eq(200)
```

Just a note again, verifying the response status is important. Otherwise, a failed response will not change the state of the notification as well.

Reply:
Yes, I usually pay attention to that but somehow I missed it in this and the previous PR. Code reviews FTW!